script/bump-version.js

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
+
+const { GitProcess } = require('dugite')
+const utils = require('./lib/version-utils')
+const plist = require('plist')
+const fs = require('fs')
+const semver = require('semver')
+const path = require('path')
+const { promisify } = require('util')
+const minimist = require('minimist')
+
+const writeFile = promisify(fs.writeFile)
+const readFile = promisify(fs.readFile)
+
+function parseCommandLine () {
+  let help
+  const opts = minimist(process.argv.slice(2), {
+    string: [ 'bump', 'version' ],
+    boolean: [ 'stable', 'dryRun', 'help' ],
+    alias: { 'version': ['v'] },
+    unknown: arg => { help = true }
+  })
+  if (help || opts.help || !(opts.bump || opts.stable)) {
+    console.log(`
+      Bump release version number. Possible arguments:\n
+        --bump=patch to increment patch version\n
+        --stable to promote current beta to stable\n
```

Comment:
Call me crazy, but I don't see where we are using `opts.stable`?

Reply:
we are...not 😛